include/tvm/relay/attrs/image.h

```diff
@@ -50,8 +51,11 @@ struct ResizeAttrs : public tvm::AttrsNode<ResizeAttrs> {
         .describe("Specify the mode to use for scaling."
                   "NEAREST_NEIGHBOR -  Nearest Neighbor"
                   "BILINEAR - Bilinear Interpolation");
```

Comment:
We have supported "BICUBIC" in this PR. I think could expose it in relay and frontend. Again, I think using uppercase is acceptable. Otherwise I think you need change many places (relay, frontend, topi and so on) to unified.